lib/api/completeMultipartUpload.js

```diff
@@ -210,8 +210,13 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 // return an error
                 const partSize = Number.parseInt(storedPartsAsObjects[i]
                     .size, 10);
-                if (i < jsonList.Part.length - 1 &&
+                // allow smaller parts for testing
+                log.info('MPU_TESTING env variable setting',
+                    { setting: process.env.MPU_TESTING });
+                if (process.env.MPU_TESTING !== 'yes' &&
+                    i < jsonList.Part.length - 1 &&
                     partSize < constants.minimumAllowedPartSize) {
+                    log.info('part too small on complete mpu');
```

Comment:
This could be a debug log or you can do you can make the log entry only for MPU_TESTING.